### PR TITLE
ONL-7324: Remove linking capabilities from ec-option-card

### PR DESCRIPTION
**Pull request by jeger-at**
As we have different ways of linking ec-option-cards we should move the way of linking to the concrete implementation in order to remove complexity from this component.
The links we had so far were
* router-link
* href
* button with js action

### CODE REVIEW

**Review by wolandec — APPROVED** [review]

File: src/components/ec-option-card/ec-option-card.vue
Comment (wolandec) [inline src/components/ec-option-card/ec-option-card.vue:1]: We should handle different types of `to` property(`string | RouteLocationPathRaw | RouteLocationNamedRaw`) and write tests for it.

File: src/components/ec-option-card/ec-option-card.vue [hunk 1]
```diff
@@ -89,20 +60,15 @@ const getOptionIconClass = computed(() => ({
...
 
-function handleClick() {
-  if (componentTag.value === 'button' && !props.isDisabled) {
-    emit(OptionCardEvent.CLICK);
-  }
-}
 </script>
 
-<style>
+<style scoped>
```
[the remark this answers is not in the document]
Comment (jeger-at) [reply src/components/ec-option-card/ec-option-card.vue:71]: good catch :)

**Comment by wolandec** [comment]
And let's rename this PR :)

**Comment by wolandec** [comment]
We need to update screenshots for this story

**Pull request merged**